index.js

```diff
@@ -5,14 +5,53 @@ console.log("Here's your header:", header)
 
 /***** Deliverable 2 *****/
 
-
+header.style.color = "#CD5C5C"
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+const h2 = document.querySelector("div#profile h2")
+h2.textContent = traveler.name
+
+
+const nickname = document.querySelector("div#profile em")
+nickname.textContent = traveler.nickname
+
+
+const photo = document.querySelector("div#profile img")
+photo.src = "/Users/yuringuyen/Flatiron/code/JavaScript/js-dom-mini-challenge/pics/raffy.jpg"
+
+
+console.log('This is what the traveler object looks like: ', traveler)
 
 
 /***** Deliverable 4 *****/
 
+function renderOneAnimalSighting(animals){
+    const li = document.createElement("li")
+    li.dataset.id = animals.id
+
+    li.innerHTML = `<li data-id="${animals.id}">
+    <p>"${animals.description}"</p>
+    <img src="${animals.photo}" alt="${animals.species}"/>
+    <a href="${animals.link}" target="_blank">Here's a video about the "${animals.species}" species!</a>
+    </li>`
+
+const ul = document.querySelector("ul#animals")
+ul.append(li)
+//  console.log(ul)
+}
+
+function renderAllAnimalsSighting(){
+
+    traveler.animalSightings.forEach(function(animals){
+        renderOneAnimalSighting(animals)
+        
+    })
+}
 
+renderAllAnimalsSighting()
```

Comment:
Good! I love that you a couple helper functions here to iterate through your array and then append the info to the DOM. This will really help in the future if you have a feature where a user can add a new element to your array. 💯